depends/patches/qt/fix_qt_pkgconfig.patch

```diff
@@ -1,11 +1,23 @@
 --- old/qtbase/mkspecs/features/qt_module.prf
 +++ new/qtbase/mkspecs/features/qt_module.prf
-@@ -245,7 +245,7 @@
+@@ -264,7 +264,7 @@
  load(qt_targets)
  
  # this builds on top of qt_common
 -!internal_module:!lib_bundle:if(unix|mingw) {
-+unix|mingw {
```

Comment:
Do you or @jonasschnelli or @theuni understand why this original patch was necessary?

Reply:
@dongcarl I don't recall, but from a quick glance, it looks like it was necessary to get qt to generate pkg-config files for internal/bundled libs (png/harfbuzz/pcre, etc).